ksqldb-engine/src/main/java/io/confluent/ksql/engine/KsqlEngine.java

```diff
@@ -174,7 +174,7 @@ public KsqlEngine(
         1000,
         TimeUnit.MILLISECONDS
     );
-    this.ksqlConfig = ksqlConfig;
+    this.ksqlConfig = Objects.requireNonNull(ksqlConfig);
```

Comment:
Use the two-arg version of `requireNonNull()` rather than the one-arg version.

Reply:
Ahh, that again. Sorry!

Reply:
This conversation was marked resolved but I don't see the change made?

Reply:
Looks like I accidentally made the change in the pull query PR instead of this one...